pusher-api/user-guides/create-subscriptions.md

```diff
@@ -172,6 +173,16 @@ To verify that events come from Zettle, calculate a signature and compare it wit
 
 2. Compare the newly calculated signature with the value in the HTTP header `X-iZettle-Signature` and take actions accordingly.
 
+## Step 5: Avoid webhook requests timeout
+Webhook requests time out 10 seconds after the Pusher API sends the events to the destination URL on your server.
+
+To avoid webhook requests failing, configure your server to handle them as soon as possible. So the destination URL can respond to the Pusher API before the timeout. A best practice is to handle webhook requests in the background.
```

Comment:
Suggest changing to: "This way the destination URL can respond to the Pusher API before the timeout occurs."

Reply:
Fixed